From 000a8684bd109c3f9c9b5fabd95add4743b7f45a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Stefan=20E=C3=9Fer?= <se@FreeBSD.org>
Date: Wed, 22 Sep 2021 17:48:07 +0200
Subject: [PATCH] Do not fail if no alias has been defined
Upstream: merged (https://github.com/pexpect/pexpect/pull/698/commits/000a8684bd109c3f9c9b5fabd95add4743b7f45a)

There is no output if no alias has been defined and the test fails in
that case. Adding an alias definition makes the alias command return
at least that just added entry.
---
 tests/test_replwrap.py | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/tests/test_replwrap.py b/tests/test_replwrap.py
index 5e50ea28..73c82e84 100644
--- a/tests/test_replwrap.py
+++ b/tests/test_replwrap.py
@@ -25,7 +25,7 @@ def tearDown(self):

     def test_bash(self):
         bash = replwrap.bash()
-        res = bash.run_command("alias")
+        res = bash.run_command("alias xyzzy=true; alias")
         assert 'alias' in res, res

         try:
@@ -92,7 +92,9 @@ def test_existing_spawn(self):
                                     "PS1='{0}' PS2='{1}' "
                                     "PROMPT_COMMAND=''")

+        print(repl)
         res = repl.run_command("echo $HOME")
+        print(res)
         assert res.startswith('/'), res

     def test_python(self):
